modules/private-clusters-about.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-private-cluster.adoc
+
+[id="private-clusters-about_{context}"]
+= About private clusters
+
+By default, {product-title} is provisioned using publicly-accessible DNS and endpoints. You can set the DNS, Ingress Controller, and API server to private after you deploy your cluster.
+
+[discrete]
+[id="private-clusters-about-dns_{context}"]
+== DNS
+
+If you install {product-title} on installer-provisioned infrastructure, the installation program creates records in a pre-existing public zone and, where possible, creates a private zone for the cluster’s own DNS resolution. In both the public zone and the private zone, the installation program creates DNS entries for `*.apps`, for ingress, and `api`, for the API server.
+
+The `*.apps` records in the public and private zone are identical, so when you delete the public zone, the private zone seamlessly provides all DNS resolution for the cluster.
+
+[discrete]
+[id="private-clusters-about-ingress-controller_{context}"]
+== Ingress Controller
+Because the default ingress object is created as public, the load balancer is internet-facing and in the public subnets. You can replace the default ingress controller with an internal one.
+
+[discrete]
+[id="private-clusters-about-api-server_{context}"]
+== API server
+
+By default, the installation program creates both a private and a public network load balancer for the API server to use. The load balances are identical except that an additional port is available on the internal one for use within the cluster. Although the installation program automatically creates or destroys the load balancer based on API server requirements, the cluster does not manage or maintain them. As long as you preserve the cluster's access to the API server, you can manually modify or move the load balancers.
```

Comment:
@e-tienne, @wmengRH, how should this information change for Azure and GCP, if at all?

Reply:
As far as I know, in 4.2 gcp install, installer only create one api load balancer for both internal and external access. That is why I said this section's description is only applicable for aws. BTW, in 4.3, IPI on gcp will introduce internal api for access inside cluster, while UPI on gcp still not.

Reply:
LGTM for Azure

The public-lb on Azure only has 1 rule for api-internal (TCP/6443) --> control-plane pool while the internal-lb also has a rule for sint (TCP/22623) --> internal-control-plane pool